alpine.Dockerfile

```diff
@@ -2,7 +2,7 @@
 #checkov:skip=CKV_DOCKER_2
 #checkov:skip=CKV_DOCKER_3
 #checkov:skip=CKV_DOCKER_7
-FROM php-base AS common
+FROM php:zts-alpine AS common
```

Comment:
This change should be reverted, this value is set in `docker-bake.hcl`: https://frankenphp.dev/docs/contributing/#building-docker-images-locally

Reply:
I comitted it by mistake while testing, should be fine now 👍